ci-operator/config/openshift/kubevirt-csi-driver-operator/openshift-kubevirt-csi-driver-operator-master.yaml

```diff
@@ -0,0 +1,43 @@
+base_images:
+  base:
+    name: "4.8"
+    namespace: ocp
+    tag: base
+  openshift_release_golang-1.13:
+    name: release
+    namespace: openshift
+    tag: golang-1.13
```

Comment:
need to change to go 1.15 - this is what's used for the project https://github.com/openshift/kubevirt-csi-driver/blob/master/go.mod

Reply:
sorry my mistake, this is the operator

Reply:
operator uses 1.13 in voth go.mod and Dockerfile